xcode/Subconscious/Shared/Components/Common/Recovery/RecoveryModeFormPanelView.swift

```diff
@@ -11,7 +11,6 @@ import ObservableStore
 
 struct RecoveryModeFormPanelView: View {
     var store: ViewStore<RecoveryModeModel>
-    var onDismiss: () -> Void
```

Comment:
We can send this action to the store instead of requiring parent to pass it down.